ublox/Cargo.toml

```diff
@@ -18,7 +18,7 @@ bitflags = "2.3.1"
 chrono = {version = "0.4.19", default-features = false, features = []}
 num-traits = {version = "0.2.12", default-features = false}
 serde = {version = "1.0.144", optional = true, default-features = false, features = ["derive"]}
-ublox_derive = {path = "../ublox_derive", version = "0.0.4"}
```

Comment:
Also we might want to use an `=` version specifier (e.g. `=0.1.0`) because the derive crate is tightly coupled to the main crate and it'll be easy to accidentally make breaking changes to it. We're only testing with the one derive version and it makes sense to only support the one.

Or maybe we just increment the 0.X.0 version on every ublox release unless we're sure the changes aren't breaking.

Reply:
>  it'll use the path when developing, but the version when you download the crate from crates.io

awesome, been looking for something like this for long 👍